### ISSUE TITLE
"Custom tabs" setting does not persist

### ISSUE THREAD

**Issue by mihe** [Mosc/Glider#148]
I just updated to 2.5.0 and it seems like every time I kill the app and start it back up this new "Custom tabs" setting has been reset to being disabled, resulting in links being opened in the external browser instead.

Let me know if there's any more information you need.

**Comment by Mosc** [comment]
Yeah, my bad. This has been fixed in 4efc867e1b41534693a663c2f85a41436dac758c on master, which will be part of the next release.
